src/create.c

```diff
@@ -169,16 +168,6 @@ crun_command_create (struct crun_global_arguments *global_args, int argc, char *
   if (container == NULL)
     libcrun_fail_with_error (0, "error loading config.json");
 
-  if (crun_context.kontain)
-    {
-      ret = add_kontain_config (container);
```

Comment:
Does moving the call to add_kontain_config() from this function: crun_command_create() into libcrun_container_create() accomplish anything?
Could you add a note in the PR description explaining the reasoning?